pom.xml

```diff
@@ -543,12 +546,26 @@
                         <groupId>org.bytedeco</groupId>
                         <artifactId>mkl-dnn-platform</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.bytedeco</groupId>
```

Comment:
This exclusion in..log4j? doesn't make any sense. Did you add this in the right place?

Reply:
No it's not in log4j, you can see the line number is different.

Reply:
Ah I see. Github's interface is weird.